ddtrace/contrib/trace_utils.py

```diff
@@ -165,6 +165,16 @@ def wrapper(wrapped, instance, args, kwargs):
     return with_mod
 
 
+def distributed_tracing_enabled(int_config, default=False):
+    # type: (IntegrationConfig, bool) -> bool
+    """Returns whether distributed tracing is enabled for this integration config"""
+    if "distributed_tracing_enabled" in int_config and int_config.distributed_tracing_enabled is not None:
+        return int_config.distributed_tracing_enabled
+    elif "distributed_tracing" in int_config and int_config.distributed_tracing is not None:
```

Comment:
Why do we use distributed_tracing_enabled and distributed_tracing keys in int_config? Is this for backwards compatibility?

Reply:
yup! for backwards compatibility reasons. we didn't unify integrations and messed up the config in the beginning.